src/pairs.rs

```diff
@@ -266,13 +266,14 @@ struct PairList<'a, 'b, T: Rewrite> {
     span: Span,
 }
 
-fn is_ident(expr: &ast::Expr) -> bool {
+fn is_ident_or_lit(expr: &ast::Expr) -> bool {
```

Comment:
nit: I think we should be more explicit about bool literals.

```suggestion
fn is_ident_or_bool_lit(expr: &ast::Expr) -> bool {
```

Might even be helpful to link to text from the style guide explaining the rules if they've been codified yet.

Reply:
they have not been codified yet, all there is to link is discussion threads :).

Reply:
All good then. No need to link to discussions.